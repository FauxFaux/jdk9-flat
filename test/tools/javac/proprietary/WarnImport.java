/*
 * @test    /nodynamiccopyright/
 * @bug     6380059
 * @summary Emit warnings for proprietary packages in the boot class path
 * @author  Peter von der Ah\u00e9
 * @compile WarnImport.java
 * @compile/fail/ref=WarnImport.out -XDrawDiagnostics  -Werror WarnImport.java
 * @compile/fail/ref=WarnImport.out -XDrawDiagnostics  -Werror -nowarn WarnImport.java
 * @compile/fail/ref=WarnImport.out -XDrawDiagnostics  -Werror -Xlint:none WarnImport.java
 */

import sun.security.x509.X509CertInfo;

public class WarnImport {}
