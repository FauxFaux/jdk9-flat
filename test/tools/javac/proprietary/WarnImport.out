WarnImport.java:12:25: compiler.warn.sun.proprietary: sun.security.x509.X509CertInfo
- compiler.err.warnings.and.werror
1 error
1 warning
